scripts/superdesk-interfaces/User.ts

```diff
@@ -26,7 +22,7 @@ export interface IUser {
     picture_url: string;
     avatar: string;
     avatar_renditions: {};
-    role: IUserRole;
+    role?: string;
```

Comment:
we just get role id, not the role itself there

Reply:
Yep, so you can add an `_id: string` or `id: string` field to `IUserRole`, and update this field to `role?: IUserRole['_id']` which compiler will understand as `string`, but it gives developers knowledge of the relationship and it will appear in "find usages" or refactorings of `IUserRole`.

Reply:
:+1: got it